Classes/Neos/Neos/Ui/Controller/BackendServiceController.php

```diff
@@ -237,10 +245,11 @@ public function discardAction(array $nodeContextPaths)
     /**
      * Change base workspace of current user workspace
      *
-     * @param string $targetWorkspaceName
+     * @param string $targetWorkspaceName,
+     * @param NodeInterface $documentNode
      * @return void
      */
-    public function changeBaseWorkspaceAction($targetWorkspaceName)
+    public function changeBaseWorkspaceAction($targetWorkspaceName, $documentNode)
```

Comment:
Parameters should be annotated `string $targetWorkspaceName, NodeInterface $documentNode`